src/RadixDlt.NetworkGateway.PostgresIntegration/Migrations/IdempotentApplyMigrations.sql

```diff
@@ -1450,3 +1450,22 @@ BEGIN
 END $EF$;
 COMMIT;
 
+START TRANSACTION;
+
+
+DO $EF$
+BEGIN
+    IF NOT EXISTS(SELECT 1 FROM "__EFMigrationsHistory" WHERE "MigrationId" = '20241211105351_NewResourceBalanceChangeTransactionMarker') THEN
+    ALTER TYPE ledger_transaction_marker_type ADD VALUE 'resource_balance_change';
+    END IF;
+END $EF$;
+
+DO $EF$
+BEGIN
+    IF NOT EXISTS(SELECT 1 FROM "__EFMigrationsHistory" WHERE "MigrationId" = '20241211105351_NewResourceBalanceChangeTransactionMarker') THEN
+    INSERT INTO "__EFMigrationsHistory" ("MigrationId", "ProductVersion")
+    VALUES ('20241211105351_NewResourceBalanceChangeTransactionMarker', '8.0.2');
```

Comment:
This doesn't include the removal of the round update transaction markers - perhaps it needs regenerating?

Reply:
good catch. done.